doc-content/enterprise-only/bpmn/bpmn-con.adoc

```diff
@@ -1,8 +1,61 @@
 [id='bpmn-con_{context}']
 = Business Process Modeling and Notation Version 2.0
-The Business Process Modeling and Notation Version 2.0 (BPMN2) specification is an Object Management Group (OMG) specification that defines standards for graphically representing a business process, defines execution semantics for the elements, and provides process definitions in XML format. 
+The Business Process Modeling and Notation Version 2.0 (BPMN2) specification is an Object Management Group (OMG) specification that defines standards for graphically representing a business process, defines execution semantics for the elements, and provides process definitions in XML format.
 
-A process is defined or determined by its process definition. It exists in a knowledge base and is identified by its ID. A  process is a container for a set of modeling elements. It contains elements that specify the execution workflow of a business process or its parts using flow objects and flows. Each process has its own BPMN2 diagram. {PRODUCT} contains the legacy process designer and the new process designer for creating BPMN2 diagrams. The new process designer has an improved layout and feature set and continues to be developed. Until all features of the legacy process designer are completely implemented in the new process designer, both designers are available in {CENTRAL} for you to use.
+A process is defined or determined by its process definition. It exists in a knowledge base and is identified by its ID.
 
+--
+.General process properties
```

Comment:
Wouldn't it be good to have this section centralized? Because pretty much the same table, as you said, is in the `Creating a business process` document [here](https://github.com/hmanwani-rh/kie-docs/pull/25/files#diff-487ba61f9459cfd6ad427c9bf15095c7R35).

Reply:
@MarianMacik, I agree with your point, and thanks for raising this. Since we didn't have general process properties documented in both documents and we cannot pull Creating a business process (procedural module) section in the BPMN reference guide, since it contains all reference modules. Therefore, these documents require streamlining, and we're working towards it.

Reply:
OK, thanks for clarification!